ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java

```diff
@@ -29,4 +62,14 @@ public final class RespConstantsV7 {
     public static final String BINOKULART_MED_KORREKTION_DELSVAR_ID_8 = "8.7";
     public static final String KONTAKTLINSER_HOGER_OGA_DELSVAR_ID_8 = "8.8";
     public static final String KONTAKTLINSER_VANSTER_OGA_DELSVAR_ID_8 = "8.9";
+
+
+    public static final String TIDIGARE_UTFORD_UNDERSOKNING_ID = "8.10";
```

Comment:
Could be good to include MESSAGE to the id-constant. Normally we have SVAR or DELSVAR. We did something similar in DB.

    public static final String POLISANMALAN_DELSVAR_ID = "7.1";
    public static final String POLISANMALAN_PREFILL_MESSAGE_DELSVAR_ID = "7.1.1";
    public static final String POLISANMALAN_PRINT_MESSAGE_DELSVAR_ID = "7.1.2";

Reply:
Great suggestion! Done